testdata/src/discard_handler/discard_handler.go.golden

```diff
@@ -0,0 +1,11 @@
+package discard_handler
```

Comment:
Could you clarify what is the purpose of the .golden file? Is it used by analysistest?

Reply:
The golden file is used by `analysistest.RunWithSuggestedFixes` to check the suggested fixes.

Reply:
TIL :)